src/r2mm/manager/ManagerSettings.ts

```diff
@@ -45,6 +45,18 @@ export default class ManagerSettings {
         await settings.save();
     }
 
+    /**
+     * Clear local settings stored in IndexedDB.
+     */
+    public static async resetLocalSettings() {
+        await ManagerSettings.DEXIE_STORE.delete();
```

Comment:
I don't like the idea that the function that's supposed to solve the potentially faulty state management of settings is itself depending on said potentially faulty state management. I'd go about the deletion in the most direct way possible with the least state management involved, i.e. the closest way comparable to the user manually deleting the IndexedDB since we know that's working.

In other words, why do we go through the abstraction here at all? Seems like a more fail-safe solution would be to drop the indexeddb directly (and without relying on the potentially buggy abstraction) & then restart the app or something along those lines

Reply:
I agree with dropping the IndexedDB directly.

I'm not sure if the restart is needed. I guess it would simplify the code, but would probably also lead into a loop where we just offer to drop the database again and again since we don't know if user has already tried that?

Reply:
I don't think the restart is necessary if it's easy to make work without it, just figured properly resetting the state might also be bug prone. In this case it seems simple enough so marking this as resolved